docs/DeveloperGuide.md

```diff
@@ -637,14 +637,16 @@ testers are expected to do more *exploratory* testing.
 
    1. Double-click the jar file Expected: Shows the GUI with a set of sample properties and appointments. The window size may not be optimum.
 
-1. Saving window preferences
+
+2. Saving window preferences
```

Comment:
wait i thought the 1. will automatically change to whatever number it is supposed to be?

Reply:
because I added a space between lines to reduce clutter, hence it doesn't

Reply:
orhhh icic